design-proposals/vpro-device.md

```diff
@@ -51,81 +48,68 @@ Let us now analyze the device activation, the user must do the following steps b
 
 ```mermaid
 sequenceDiagram
-    title AMT provisioning through Tinker action
+    title DMT provisioning through Agent
```

Comment:
Add block for orchestrator components and EN components to separate out components